webpages/settings/index.html

```diff
@@ -299,7 +306,7 @@ <h1 v-cloak>{{ msg("settings") }}</h1>
         </div>
       </div>
     </div>
-    <modal class="more-settings" :is-open.sync="moreSettingsOpen" :title="msg('moreSettings')" v-cloak>
+    <modal class="more-settings" :is-open.sync="moreSettingsOpen" :title="msg('moreSettings')" v-el:moreSettings>
```

Comment:
The `v-cloak` attribute should stay - otherwise the content of the modal can briefly appear at the bottom of the popup after switching from a different tab to Addons:

<img width="442" height="645" alt="Image" src="https://github.com/user-attachments/assets/1dee962a-d508-4de1-b001-3b2ab999f7d3" />